docs/user_guide/service_config.md

```diff
@@ -720,6 +721,17 @@ Example:
 ```yaml
 health.checkInterval: "1m"
 ```
+
+#### event.RESTResponseSize
+Sets the maximum number of events that are returned by the batch event API (`/ws/v1/events/batch`).
```

Comment:
Should we say something about why there is no similar config for stream?

Reply:
Not really, streaming is fundamentally different. This is only relevant for batches.